### ISSUE TITLE
searching software version with space fails

### ISSUE THREAD

**Issue by inphobia** [netdisco/netdisco#647]
## Expected Behavior

search with spaces needs to be reviewed. mostly reminder for myself.

## Current Behavior

searching for software versions which contain a space returns no results. currently we see this on pulse-secure devices with os `8.3R7.1 (build 65025)`

## Possible Solution

either fix the os version query in netdisco or make snmp::info return os versions without spaces.

## Steps to Reproduce (for bugs)

1. search for os version with spaces: `/search?tab=device&q=ive&dns=&ip=&name=&location=&description=&os=ive&os_ver=8.3R7.1+(build+65025)&matchall=on`
2. os name does seem to work with spaces: `VMware ESXi` returns results

## Your Environment

* Netdisco version used: master
* SNMP::Info version used: master

**Comment by ollyg** [comment]
Hi @inphobia I cannot reproduce this.
I ran a query on my system where there is a Net-SNMP device, which has version "18.5.0 / Net-SNMP 5.6.2.1":
````
/search?tab=device&q=ive&dns=&ip=&name=&location=&description=&os_ver=18.5.0+/+Net-SNMP+5.6.2.1&matchall=on
````
I even changed the data in my local system:
````
netdisco=# update device set os_ver = '8.3R7.1 (build 65025)' where os_ver = '18.5.0 / Net-SNMP 5.6.2.1';
UPDATE 1
````
And still it returns the result.
Could it be some other bug such as bad characters in the os_ver ?? What is in the database field?

**Comment by inphobia** [comment]
good catch, after review it seems the pulse secure appliance add a newline to it's os version.

i wonder what's the best place to fix this snmp::info seems logical, but perhaps netdisco could also do more input validation? pro's & cons to both.

lmk where you think is most appropriate & i'll patch it.

**Comment by inphobia** [comment]
fixed in netdisco/snmp-info#364
